spec/models/post_tag_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: post_tags
+#
+#  id         :bigint           not null, primary key
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  post_id    :bigint
+#  tag_id     :bigint
+#
+# Indexes
+#
+#  index_post_tags_on_post_id             (post_id)
+#  index_post_tags_on_post_id_and_tag_id  (post_id,tag_id) UNIQUE
+#  index_post_tags_on_tag_id              (tag_id)
+#
+require 'rails_helper'
+
+RSpec.describe PostTag, type: :model do
+  let(:post_tag) { create(:post_tag) }
+
+  it 'tag must exist' do
+    Tag.find(expect(post_tag.tag_id)).exist
```

Comment:
This is wrong. Find https://relishapp.com/ RSpec examples.
Also there's no sense to do extra query like `Tag.find(...)`
This might look like
```
expect(post.tag).to exist
```
Also, it would be nice to have test cases where we can pass specific a tag and a post and see if it was stored correctly.
```
let(:tag) { create(:tag) }
let(:post) { create(:post) }
let(:post_tag) { create(:post_tag, post: post, tag: tag) }

it { expect(post_tag.post).to eq(post) }
it { expect(post_tag.tag).to eq(tag) }
```

Similar checks should be done for user interests